src/app/clusters/power-source-server/power-source-server.cpp

```diff
@@ -55,6 +55,10 @@ CHIP_ERROR PowerSourceAttrAccess::Read(const ConcreteReadAttributePath & aPath,
         // TODO: Needs implementation.
         err = aEncoder.EncodeEmptyList();
         break;
+    case EndpointList::Id:
+        // TODO: Needs implementation and a way to allow dynamic endpoints to register endpoints
+        err = aEncoder.EncodeEmptyList();
```

Comment:
I would rather we returned an error here rather than encoding incorrect data, for now.  But I guess that's hard if we want to update the YAMLs?

We just need to make sure we fix this before we ship.  Is there a tracking issue?

Reply:
This isn't incorrect - this is the value for "entire node". Issue here: https://github.com/project-chip/connectedhomeip/issues/27995. No project set up for this stuff yet, but I'll try to get that done this afternoon.

Reply:
Ah, makes sense.  OK, thanks!